client/src/components/common/Button/ContainedButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
```

Comment:
you don't have to import Button from material UI cause we want to create our own here , so delete this from here